src/components/pages/SideDrawer/_SideDrawer.less

```diff
@@ -86,21 +131,28 @@ li {
   }
 }
 
+// l.drop-down-item {
+//   padding-right: 0 !important;
+// }
+
 .ant-menu-item-selected {
-  color: @selected-color;
+  color: @gray;
+  background-color: @selected-color !important;
```

Comment:
Looks like you figured out how to get active status styling working on the menu - that's great!